### PR TITLE
fix(iOS): update to 0.5.1 and screenshots

### PR DESCRIPTION
**Pull request by JimmyMow**
Update the iOS tutorial

### CODE REVIEW

File: iOS-remote-node-setup.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
 Just to make sure everything went ok we can run `go version` to ensure we have go 1.10 installed:
```
Comment (mrfelton) [inline iOS-remote-node-setup.md]: Should say "to ensure we have go 1.11 installed"

**Review by mrfelton — CHANGES_REQUESTED** [review]
missed one change of `1.10` to `1.11`

File: iOS-remote-node-setup.md (hunk 1 above)
Comment (JimmyMow) [reply iOS-remote-node-setup.md]: Fixed @mrfelton

**Comment by JimmyMow** [comment]
@mrfelton can you re-review?

**Review by mrfelton — APPROVED** [review]

**Pull request merged**